account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  custom,
+  keccak256,
+  parseEther,
+  publicActions,
+  toHex,
+  type Address,
+} from "viem";
+import { privateKeyToAccount } from "viem/accounts";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'privateKeyToAccount' is defined but never used.
<!-- __reviewdog__:ChA1YzY0ODhiZTg3NmE4NTgxEgZyZGpzb24= -->